gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -336,15 +339,15 @@ If `flex-gateway.service` has a status of `active`, Flex Gateway is successfully
 
 // end::k8s-install-flex-helm-chart-title[]
 // tag::k8s-install-flex-helm-chart-intro-connected[]
-Use Helm to deploy Flex Gateway to a node in your cluster and connect to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+
+Register Flex Gateway, and use Helm to deploy Flex Gateway to a node in your cluster and connect to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+
 // end::k8s-install-flex-helm-chart-intro-connected[]
 // tag::k8s-install-flex-helm-chart-intro-local[]
-Use Helm to deploy Flex Gateway to a node in your cluster. 
+
+Register Flex Gateway, and use Helm to deploy Flex Gateway to a node in your cluster. 
 // end::k8s-install-flex-helm-chart-intro-local[]
-// PLS don't move k8s-mode-overview
```

Comment:
we removed flex-gateway-k8-overview.adoc[]; this is no longer needed bc the content on that page was moved to the pages where this include resided